FirebaseAuth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 10.13.0
+- [added] Added reCAPTCHA verification support in email authentication flows. (#11231)
+- [fixed] Fixed a bug where `UIApplication.shared` was accessed in iOS app extensions. (#11537)
```

Comment:
Fixed a compilation error regression introduced in 10.12.0 building iOS App Extensions. (#11537)

Reply:
Thanks for the suggestion, Paul! I removed the line for now and we can add the entry in #11537 if it resolves the issue.